src/librbd/ObjectMap.cc

```diff
@@ -165,8 +165,14 @@ void ObjectMap<I>::close(Context *on_finish) {
     return;
   }
 
-  auto req = object_map::UnlockRequest<I>::create(m_image_ctx, on_finish);
-  req->send();
+  auto ctx = new FunctionContext([this, on_finish](int r) {
```

Comment:
> on_finish vs ctx

That is the big reference counting changeset missing in nautilus, not a simple rename.  Just want to make sure that delaying the object map unlock doesn't depend on that in any way?

Reply:
I can't think of any reasons why this close delay would make the situation worse.